### DISCUSSION TITLE
How to hide the TabBar customise  ?

### DISCUSSION THREAD

**Discussion by FelixFalkovsky** [rundfunk47/stinsen#117] [General]
How to hide the TabBar customise  ? For IOS 14+

**Comment by LePips** [comment]
Do you mean hiding the tab bar? In iOS 16, you can use `.toolbar(.hidden, for: .tabBar)`.

**Reply by FelixFalkovsky** [reply]
IOS 14+

**Reply by LePips** [reply]
Then you can try introspecting. Regardless, this is a SwiftUI question and not a Stinsen one.

**Comment by 2jumper3** [comment]
@LePips can u help with hide tabbar, when I used customise func in Stinsen? 
`
 @ViewBuilder func customize(_ view: AnyView) -> some View { 
 
        ZStack(alignment: .bottom) {
            view
                .overlay(alignment: .bottom) {
                    makeBars()
                        .offset(y: 7)
                }
        }.environment(\.activeTab, self.child.activeTab)
            .toolbar(.hidden, for: .navigationBar)
            .ignoresSafeArea(.keyboard, edges: .bottom)
    }
`

This overlay always showing, even when .toolbar(.hidden)

**Reply by est7** [reply]
any updates?
